addons/project_mrp/models/mrp_production.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
```

Comment:
Not necessary anymore, the encoding defaults to utf-8 in python 3 :slightly_smiling_face: